plugins/topology/src/models.ts

```diff
@@ -74,3 +74,59 @@ export const resourceModels: { [key: string]: GroupVersionKind } = {
   [ModelsPlural.jobs]: JobGVK,
   [ModelsPlural.statefulsets]: StatefulSetGVK,
 };
+
+export const DeploymentModel = {
```

Comment:
the purpose for bringing in `...GVK` was to go away from models as used in the console based on the suggestion from Sam. Do we need all of these or can we compute some on the fly?

Reply:
yeah if needed good to keep only those that are needed and if we need other properties then we don't need to have `..GVK` and `...models` both can extend the existing ones themselves.

@spadgett Let us know if you see any issues with this.